<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Area</class>
 <widget class="QWidget" name="Area">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>701</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="3">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>FIPS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Area_Name</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Display">
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="MainPage">
        <property name="text">
         <string>MainPage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Population">
        <property name="text">
         <string>Population</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="County">
        <property name="text">
         <string>County</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Covid_Cases">
        <property name="text">
         <string>Covid_Cases</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="State">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="box3">
      <item>
       <property name="text">
        <string>Area_Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FIPS</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="box1">
      <item>
       <property name="text">
        <string>Area_Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FIPS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ALL</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Search">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>Please type Area_Name or FIPS</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="box2">
      <item>
       <property name="text">
        <string>ALL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>First 10 Rows</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>First 100 Rows</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
